From 8b0efca4050a385df604d2279145a93b5e6f8153 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 30 Oct 2018 17:51:20 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: Change tsadc clock rate to
 650KHz

The clock frequency should be between 500KHz and 800KHz, 650KHz is
a typical value.

Change-Id: Id8a81f667350747576f803ce5259b4e09076be89
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index ec36f2a81002..e7a065133473 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -435,7 +435,7 @@
 		clocks = <&cru SCLK_TSADC>, <&cru PCLK_TSADC>;
 		clock-names = "tsadc", "apb_pclk";
 		assigned-clocks = <&cru SCLK_TSADC>;
-		assigned-clock-rates = <50000>;
+		assigned-clock-rates = <650000>;
 		resets = <&cru SRST_TSADC>;
 		reset-names = "tsadc-apb";
 		#thermal-sensor-cells = <1>;
-- 
2.35.3

